clang/runtime/dpct-rt/include/dpct/bindless_images.hpp

```diff
@@ -0,0 +1,688 @@
+//===--- bindless_images.hpp ----------------------*- C++ -*---------------===//
+//
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __DPCT_BINDLESS_IMAGE_HPP__
+#define __DPCT_BINDLESS_IMAGE_HPP__
+
+#include <sycl/sycl.hpp>
+
+#include "image.hpp"
```

Comment:
Please have image.hpp to include bindless_images.hpp.

Reply:
fixed